sde_collections/models/collection_choice_fields.py

```diff
@@ -97,3 +97,56 @@ class WorkflowStatusChoices(models.IntegerChoices):
     PROD_MAJOR = 16, "Prod: Major Issues"
     MERGE_PENDING = 17, "Code Merge Pending"
     NEEDS_DELETE = 19, "Delete from Prod"
+
+
+class TDAMMTags(models.TextChoices):
+    """TDAMM (Tagged Data for Multi-Messenger Astronomy) tag choices."""
+
```

Comment:
add a choice called NOT_TDAMM = "Not TDAMM", "Not TDAMM"
This will be the default choice in the models.

Reply:
Do this and make a new migration.